ci-operator/jobs/kubevirt/common-templates/kubevirt-common-templates-master-presubmits.yaml

```diff
@@ -15,8 +15,8 @@ presubmits:
       ci-operator.openshift.io/cloud-cluster-profile: azure4
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-kubevirt-common-templates-master-e2e-azure-common-templates-centos6
-    rerun_command: /test e2e-azure-common-templates-centos6
+    name: pull-ci-kubevirt-common-templates-master-e2e-azure-common-templates-centos-stream8
+    rerun_command: /test e2e-azure-common-templates-centos-stream8
     spec:
```

Comment:
Is this removing test for the centos6 template?

Reply:
No I am not removing centos 6, 7 jobs. I am removing only centos 8 job and adding centos stream 8, 9 jobs. The script which generates these manifests just moved the centos 6, 7 jobs to a bottom (see https://github.com/openshift/release/pull/24986/files#diff-af43041056e5502cbb0dfa0da46c6b109c227d765d96b10e499d64a6ac5d7e5dR224)

Reply:
I got it, thanks.